third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -472,84 +489,82 @@ void initNvFuserPythonBindings(PyObject* module) {
-      op_str,                                                                 \
-      [](FusionDefinition::Operators& self,                                   \
-         Scalar arg1,                                                         \
-         Tensor arg2) -> Tensor {                                             \
-        FUSER_PERF_SCOPE("Operators." op_str);                                \
-        TORCH_CHECK(                                                          \
-            self.validUse(), "Attempting to add to a completed definition!"); \
-        FusionDefinition* fd = self.fusion_definition;                        \
-        Tensor output = fd->defineTensor(arg2.dims);                          \
-        fd->defineRecord(new OpRecord<TensorView*, Val*, TensorView*>(        \
-            {fd->recordingState(arg1()), fd->recordingState(arg2())},         \
-            {fd->recordingState(output())},                                   \
-            ("ops." op_str),                                                  \
-            static_cast<                                                      \
-                                                                              \
-                TensorView* (*)(Val*, TensorView*)>(op_name)));               \
-        return output;                                                        \
-      },                                                                      \
-      py::return_value_policy::reference);                                    \
-  nvf_ops.def(                                                                \
-      op_str,                                                                 \
-      [](FusionDefinition::Operators& self,                                   \
-         Scalar arg1,                                                         \
-         Scalar arg2) -> Scalar {                                             \
-        FUSER_PERF_SCOPE("Operators." op_str);                                \
-        TORCH_CHECK(                                                          \
-            self.validUse(), "Attempting to add to a completed definition!"); \
-        FusionDefinition* fd = self.fusion_definition;                        \
-        Scalar output = fd->defineScalar();                                   \
-        fd->defineRecord(new OpRecord<Val*, Val*, Val*>(                      \
-            {fd->recordingState(arg1()), fd->recordingState(arg2())},         \
-            {fd->recordingState(output())},                                   \
-            ("ops." op_str),                                                  \
-            static_cast<Val* (*)(Val*, Val*)>(op_name)));                     \
-        return output;                                                        \
-      },                                                                      \
+#define NVFUSER_PYTHON_BINDING_BINARY_OP(op_str, op_name)                      \
```

Comment:
a. git at its best :angry:

Reply:
The change here is to add the appropriate `serde::RecordType` Enum to the `OpRecord` constructor.